userspace/libsinsp/logger.h

```diff
@@ -117,7 +117,14 @@ class SINSP_PUBLIC sinsp_logger
 	 *
 	 * Note that this is intentionally inline.
 	 */
-	bool is_enabled(const severity sev) const { return (sev <= m_sev); }
+	bool is_logging(const severity sev) const { return (sev <= m_sev); }
```

Comment:
I'd avoid this API break!

Reply:
It would also greatly reduce the size of this pr.

Reply:
uhm ok, the name is not so clear to me, but I reverted the changes :)